packages/components/src/Tree/TreeItem.tsx

```diff
@@ -177,6 +188,8 @@ const TreeItemLayout: FC<TreeItemProps> = ({
     </HoverDisclosure>
   )
 
+  const TextWrapper = truncate ? Truncate : Identity
```

Comment:
Named `[React].Fragment` can be used here instead.

```suggestion
  const TextWrapper = truncate ? Truncate : Fragment
```

Reply:
oh that's awesome!